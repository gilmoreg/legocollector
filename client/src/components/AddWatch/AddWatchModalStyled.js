import styled from 'styled-components';

export const StyledHeader = styled.header`
  margin: 0;
  font-size: 32px;
  font-weight: bold;
`;

export const StyledInput = styled.input`
  width: 200px;
  margin-bottom: 1rem;
  text-align: center;
  border-radius: 5px;
  background-color: #CCCCCC;
`;

export const StyledSmall = styled.small`
  font-size: 14px;
  color: #FF7F66;
  display: block;
  margin-bottom: 0.5rem;
`;