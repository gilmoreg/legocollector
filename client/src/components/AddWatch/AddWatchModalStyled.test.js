import React from 'react';
import { shallow } from 'enzyme';
import { StyledHeader, StyledInput, StyledSmall } from './AddWatchModalStyled';

describe('Test styled components for AddWatchModal', () => {
  it('StyledHeader renders without crashing', () => {
    const wrapper = shallow(
      <StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });

  it('StyledInput renders without crashing', () => {
    const wrapper = shallow(
      <StyledInput />);
    expect(wrapper).toMatchSnapshot();
  });

  it('StyledSmall renders without crashing', () => {
    const wrapper = shallow(
      <StyledSmall />);
    expect(wrapper).toMatchSnapshot();
  });
});