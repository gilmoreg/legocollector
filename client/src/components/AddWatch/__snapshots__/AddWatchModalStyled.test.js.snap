// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test styled components for AddWatchModal StyledHeader renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "className": "sc-bdVaJa cKvjqZ",
    },
    "ref": null,
    "rendered": null,
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "className": "sc-bdVaJa cKvjqZ",
      },
      "ref": null,
      "rendered": null,
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Test styled components for AddWatchModal StyledInput renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.input />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "className": "sc-bwzfXH bmYyXl",
    },
    "ref": null,
    "rendered": null,
    "type": "input",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "className": "sc-bwzfXH bmYyXl",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Test styled components for AddWatchModal StyledSmall renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.small />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "className": "sc-htpNat czXwuo",
    },
    "ref": null,
    "rendered": null,
    "type": "small",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "className": "sc-htpNat czXwuo",
      },
      "ref": null,
      "rendered": null,
      "type": "small",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
